src_c/polygon.c

```diff
@@ -341,7 +367,90 @@ pg_polygon_copy(pgPolygonObject *self, PyObject *_null)
     return pgPolygon_New(&self->polygon);
 }
 
+static PyObject *
+pg_polygon_move(pgPolygonObject *self, PyObject *const *args, Py_ssize_t nargs)
+{
+    double x = 0, y = 0;
```

Comment:
i would say call then Dx, Dy and not just x, y
as i see it with x, y you will be setting the coordinates but with Dx, Dy you are calculating some difference

Reply:
I already changed whole function so no worries there.